test/automation/navigation.tests.ps1

```diff
@@ -1209,7 +1209,132 @@ AfterEach{
         test-path .\Bill\BITS | Should be $True
         test-path .\Bill\BITS -Type Container | Should be $False
         test-path .\Bill\BITS -Type Leaf | Should be $True
-    }
+     }
        
  }
+
+Describe "Not Supported Commands test" -Tags "Feature"{
+BeforeEach{
```

Comment:
I find the indentation in this file odd. You are just matching the style, so there is no change required by my comment, but this is what I would expect:

```powershell
Describe "" {
    BeforeEach {
        … BeforeEach content
    }

    It "" {
        … test code
    }
}
```
Each internal block (scriptblock) is indented four spaces.

Reply:
Yeah, I found that too. Current layout makes indention weird in VisualStudioCode.
After this code change, we can make a format only PR.